app/src/main/java/org/mozilla/fenix/compose/MessageCard.kt

```diff
@@ -9,19 +9,20 @@ import androidx.compose.foundation.background
 import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
```

Comment:
I am curious do you know if there's a lint rule that would enforce proper sorting of imports?

Reply:
I can check, but I'm getting this for free from just doing the Android Studio import clean-up shortcut

Reply:
Ya, I understand we get this for free. My concern is around the fact that we're checking in code that are not properly formatted, which means we end up having to do this clean up later when I think we would all prefer if this kind of stuff is all in order from the get-go.

Reply:
Don't feel you need to take too much time to dig into this. I meant it as an immediate yes or no question. If it was an immediate yes and we knew how, I would probably just get that turned on so we can clean up all our imports.

Reply:
@MozillaNoah Nice! I assigned myself to it. Thank you for taking the time to look into it.